### PR TITLE
ARKV6X RC IN USART6 4.5 Backport

### PR DESCRIPTION
**Pull request by AlexKlimaj**
Backport of @dakejahl work in master to fix SBUS input on the RC IN port.

### CODE REVIEW

File: libraries/AP_HAL_ChibiOS/hwdef/ARKV6X/hwdef.dat [hunk 1]
```diff
@@ -81,7 +85,8 @@ PD9 USART3_RX USART3
 
 # USART6 is for IOMCU
 PC6 USART6_TX USART6
-PC7 USART6_RX USART6
+# used for RC SBUS
+#PC7 USART6_RX USART6
```
Comment (tridge) [inline libraries/AP_HAL_ChibiOS/hwdef/ARKV6X/hwdef.dat:89]: happy to bring this in, but I'm curious why SBUS didn't work on this UART? Did the user set SERIAL8_PROTOCOL to RCIN?

Comment (peterbarker) [reply libraries/AP_HAL_ChibiOS/hwdef/ARKV6X/hwdef.dat:89]: Ping @dakejahl - unanswered question here.

Comment (dakejahl) [reply libraries/AP_HAL_ChibiOS/hwdef/ARKV6X/hwdef.dat:89]: The conversation is in a now deleted slack workspace but I think ultimately the issue ended up being the user had bad hardware. I can test again to be sure. Are there concerns with this change?

Comment (peterbarker) [reply libraries/AP_HAL_ChibiOS/hwdef/ARKV6X/hwdef.dat:89]: > The conversation is in a now deleted slack workspace but I think ultimately the issue ended up being the user had bad hardware. I can test again to be sure. Are there concerns with this change?

I think tridge was just curious.  He'll see your response as part of the backport process.

Comment (dakejahl) [reply libraries/AP_HAL_ChibiOS/hwdef/ARKV6X/hwdef.dat:89]: I'm not as familiar with ardupilot so if something doesn't look right let me know. Thanks for the review

**Comment by tridge** [comment]
this does seem a strange change, what testing has been done?

**Comment by dakejahl** [comment]
I tested a few months ago and it was working. I can't remember everything I did and I can't seem to figure it out from the documentation. SBUS is not working at all for me using 4.5.5 or master (with or without this PR)

SERIAL8_PROTOCOL set to RCIN
RC_PROTOCOL set to 1 (all)

Am I missing something?


I even went back and tried 4.3.3 since this was when the board support was first introduced and SBUS worked back then (I checked the logs alex shared) but it's still not working.
https://github.com/ArduPilot/ardupilot/pull/21865

I am testing on a variant of this board with an IIM42653 IMU which does not yet have support in ardupilot, so I am getting a constant error about "failed to initialize INS", could this be interfering with the SBUS setup somehow?

I know the hardware is working because when I flash to PX4 it works.

**Comment by peterbarker** [comment]
Closing per directions in #28015

**Comment by rmackay9** [comment]
I've removed the 4.5 backport project because it doesn't look like we want to do that

**Pull request closed without merging**